docs/manual/nullness-checker.tex

```diff
@@ -122,6 +122,12 @@
   checker will unsoundly assume that the argument to \<Map.get> is a key
   for the receiver map.  It will not do any checking of
   \refqualclass{checker/nullness/qual}{KeyFor} and related qualifiers.
+
+\item
+  \label{nullness-conservativeArgumentNullnessAfterInvocation}%
+  If you supply the \code{-AconservativeArgumentNullnessAfterInvocation} command-line option, then the
```

Comment:
Update this text similarly to the changelog changes I suggested.

Reply:
fixed